### PR TITLE
Refs [TASK][35588] Handle event Detail Category Screen

### PR DESCRIPTION
**Pull request by zGararz**
## Related Tickets
- [#Task 35588: Handle event Detail Category Screen](https://edu-redmine.sun-asterisk.vn/issues/35588)
## WHAT
- 
## Review Checklist

Category | View Point | Description | Self review | Reviewer2 (name)
--- | --- | --- | --- | ---
Conventions | Sun* coding conventions | https://github.com/framgia/coding-standards/tree/master/eng/android |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Conventions | Kotlin coding conventions | https://kotlinlang.org/docs/coding-conventions.html |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Sun* Redmine working process  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md |<li>- [ ] yes</li>|<li>- [ ] yes</li>  

## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/example/itbook/ui/detailcategory/DetailCategoryFragment.kt [hunk 1]
```diff
@@ -1,17 +1,85 @@
 package com.example.itbook.ui.detailcategory
 
+import android.util.Log
```
Comment (huytq-2422) [inline app/src/main/java/com/example/itbook/ui/detailcategory/DetailCategoryFragment.kt]: self review

File: app/src/main/java/com/example/itbook/ui/detailcategory/DetailCategoryFragment.kt [hunk 2]
```diff
@@ -1,17 +1,85 @@
...
+import com.example.itbook.data.source.local.BooksLocalDataSource
+import com.example.itbook.data.source.local.Database.BooksDatabase
+import com.example.itbook.data.source.local.dao.BookDaoImp
+import com.example.itbook.data.source.remote.BookRemoteHandler
+import com.example.itbook.data.source.remote.BooksRemoteDataSource
+import com.example.itbook.ui.adapter.PreviewBookAdapter
+import com.example.itbook.ui.dialog.LoadingDialogFragment
+import kotlinx.android.synthetic.main.fragment_detail_category.*
+
+class DetailCategoryFragment(private val category: String) : BaseFragment(),
```
Comment (huytq-2422) [inline app/src/main/java/com/example/itbook/ui/detailcategory/DetailCategoryFragment.kt]: em truyền qua bundle chứ

File: app/src/main/java/com/example/itbook/ui/detailcategory/DetailCategoryFragment.kt [hunk 3]
```diff
@@ -1,17 +1,85 @@
...
+        presenter?.start()
     }
 
     override fun initListeners() {
+        imageBackDetailCategory.setOnClickListener{ onBackClickListener() }
+    }
+
+    override fun showBooks(books: List<Book>) {
+        val booksSize = this.books.size - 1
+        val insertCount = books.size - this.books.size
```
Comment (huytq-2422) [inline app/src/main/java/com/example/itbook/ui/detailcategory/DetailCategoryFragment.kt:56]: sao k viết trên 1 dòng luôn

Comment (zGararz) [reply app/src/main/java/com/example/itbook/ui/detailcategory/DetailCategoryFragment.kt:56]: làm sao để viết 2 dòng này trên cùng một dòng được ạ?

File: app/src/main/java/com/example/itbook/ui/detailcategory/DetailCategoryFragment.kt (hunk 2 above)
Comment (zGararz) [reply app/src/main/java/com/example/itbook/ui/detailcategory/DetailCategoryFragment.kt]: dạ vâng để em sửa

**Pull request merged**